.travis/build-pkgs

```diff
@@ -8,8 +8,9 @@ set -eu
 # changed/created in the specified revisions.
 changed_repos() {
 	local commit_ish="$1"
+	local files="$(find -type f -maxdepth 1 | sed 's:^\./::' | tr '\n' '|' | sed 's:|$::')"
 
-	git diff-tree --name-only "$commit_ish" | grep -v '^\..*' || echo ''
+	git diff-tree --name-only "$commit_ish" | grep -Ev -e '^\..*' -e $files || echo ''
```

Comment:
Adding the `-E` flag is needed since the variable `$files` now contains escaped (`|`) pipes, instead of the escaped pipes (`\|`) the previous expression produced.